air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -95,6 +132,11 @@ proc.compute_evaluate_boundary_constraints
     padw mem_loadw.4294900205 movdn.3 movdn.3 drop drop ext2mul
 end # END PROC compute_evaluate_boundary_constraints
 
+# Procedure to compute the integrity constraint.
```

Comment:
minor nit: should this be "Procedure to evaluate all integrity constraints".

Reply:
changed